humble/distribution.yaml

```diff
@@ -6885,12 +6885,15 @@ repositories:
       version: humble
     release:
       packages:
+      - lifecyclenode_client
```

Comment:
Are these package additions intentional? They do not adhere to REP-144 guidelines - but from what I can tell they are sample code within the SMACC repository, not intended for release.

Reply:
Yes, they are intentional. In what way do they not adhere to REP-144? These package additions are SMACC Client Behavior Libraries, and serve as modular adapters to different ROS systems. We have clients for Nav2, MoveIt, these two and a few more https://github.com/robosoft-ai/SMACC2/tree/humble/smacc2_client_library. These two offer functionality used by some sample state machines such as sm_atomic.

Reply:
Looks good to me, not sure which naming rule was violated.

Reply:
`lifecyclenode_client` and `ros_timer_client` both sound really generic - but these subpackages are specific to `smacc2`, are they not? That's what I'm referring to about [REP-144](https://ros.org/reps/rep-0144.html)

> Package names should be specific enough to identify what the package does

Perhaps more appropriate names for these packages would use `smacc2` prefixing: `smacc2_lifecycle_client`, `smacc2_timer_client`

> A package name should not contain `ros` as it is redundant

This is my concern with `ros_timer_client`.

Reply:
Makes sense, [nav2_lifecycle_manager ](https://index.ros.org/p/nav2_lifecycle_manager/github-ros-planning-navigation2/#humble) is probably a good example here.

Reply:
Hello. I introduced some changes: only smacc2 and smacc2_msgs are now distributed by the moment